### ISSUE TITLE
Input is not AA accessible for screen readers

### ISSUE THREAD

**Issue by ellieCrome** [mbenford/ngTagsInput#811]
The input created when you use ng-tags does not have an aria-label or aria-labelledby attribute. I cannot see a way to add this manually in my app as the input is created through this direction. 

It would be great to have a new param available in the directive so that we can apply either of the attributes above
